.github/workflows/pr.yaml

```diff
@@ -52,6 +52,17 @@ jobs:
         timeout-minutes: 5
         run: |
           make kind-load-apiserver deploy-example-apiserver KIND_NAME="chart-testing"
+          until kubectl wait --timeout=2m --for=condition=ready pod -l app.kubernetes.io/component=apiservice,app.kubernetes.io/instance=example-konkservice
+          do
+            sleep 1
+          done
```

Comment:
You can remove this, the helm command uses `--wait` which implements a more complete version of this ready pod check here.